### PR TITLE
DEVELOPER-3536 - Add style for background on microservices topic

### PR DESCRIPTION
**Pull request by KamiQuasi**
https://issues.jboss.org/browse/DEVELOPER-3536

Blue area should have the appropriate background.

### CODE REVIEW

File: stylesheets/_topics.scss [hunk 1]
```diff
@@ -410,6 +410,12 @@ ul.topic-resources.topic-resources-list{
   max-width: 85%;
 }
 
+// Style for custom microservices topic page
+.main-promo.microservices {
+  background-image: url(http://static.jboss.org/_root/images/rhd/topic/RHDev_topics_primopromo_microservicesicon_20sept2016.png);
+  background-size: cover;
+}
+
```
Comment (paulrobinson) [inline stylesheets/_topics.scss]: Shouldn't this go with its counterparts in https://github.com/KamiQuasi/developers.redhat.com/blob/36b9d4f5576116646a24cbf0a6586c7dd6f22b94/stylesheets/_topics.scss#L83-L108

**Comment by paulrobinson** [comment]
I'm not seeing it on https://developers-pr.stage.redhat.com/pr/1726/export/microservices/

**Comment by KamiQuasi** [comment]
@paulrobinson made a poor assumption about the location of the SASS "cdn" function's whereabouts. Fixed the URL reference now, and should be good.

**Comment by KamiQuasi** [comment]
retest this please

File: stylesheets/_topics.scss [hunk 2]
```diff
@@ -100,6 +100,10 @@
       background-image: cdn("../images/design/topics/dotted_lines_white.png"), cdn('../images/design/topics/RHDev_topics_primopromo_mobile.png');
       display: block;
     }
+    &.microservices {
+      background-image: url(http://static.jboss.org/_root/images/rhd/topic/RHDev_topics_primopromo_microservicesicon_20sept2016.png);
```
Comment (paulrobinson) [inline stylesheets/_topics.scss]: should be https

**Comment by robpblake** [comment]
@paulrobinson / @KamiQuasi This one good to go?

cheers,

Rob

**Comment by KamiQuasi** [comment]
@robpblake I think so, I've resolved the couple things that @paulrobinson pointed out

**Pull request merged**